docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge link="https://strapi.io/pricing"/> <SilverBadge link="https://strapi.io/pricing"/> <GoldBadge link="https://strapi.io/pricing" withLinkIcon/>
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators. Their roles and permissions are [configured in the admin panel](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). In addition to the 3 default roles (Author, Editor, and Super Admin), it's possible to create custom conditions to match specific needs of roles and permissions management. This requires an Enterprise Edition with at minimum a Bronze licence plan.
+
+#### Defining a new condition
+
+Conditions are defined as an array of objects. Each condition object can have 4 possible properties:
+
+- `displayName` (string): the condition name as showed in the admin panel
+- `name` (string): the condition name, kebab-cased
```

Comment:
To confirm with @Convly, I think this is a unique name. We may say this detail. The name behaves as an unique id

Reply:
Not unique regarding all conditions, but unique for every condition within its namespace (either the plugin attribute or the root namespace if `plugin` is not defined)

Reply:
Basically, we generate an identifier for every action based on its `name` & `plugin` with the following logic:

- `plugin` is not defined => application::`name`
- `plugin` is defined & its value is `admin` => admin::`name`
- `plugin` is defined & its value is not `admin` => plugins::`plugin`.`name`

So if I have a condition A `{ plugin: 'foo', name: 'A' }` and a condition B `{ plugin: 'bar', name: 'A' }`, then it's all good (since the IDs will be differents).
However, if I have a condition A `{ plugin: 'foo', name: 'A' }` and a condition B `{ plugin: 'foo', name: 'A' }`, then there will be a conflict and one of the condition will erase the other.

Reply:
Thank you so much!
I'll try to keep it clear & simple in the docs but I'm making a note of your detailed answer. 
Or maybe I'll add a TIP callout with this 🤔 Will think about it a bit more!
Anyway, a thousand time thanks! 🙏